database/src/main/java/com/firebase/ui/database/FirebaseListAdapter.java

```diff
@@ -147,4 +152,5 @@ public View getView(int position, View view, ViewGroup viewGroup) {
      */
     abstract protected void populateView(View v, T model, int position);
 
+    abstract protected void onCancelled(DatabaseError databaseError);
```

Comment:
note: please add a JavaDoc to this method so that users overriding it understand what it means.